engine/src/main/scala/cromwell/logging/TerminalLayout.scala

```diff
@@ -22,25 +24,23 @@ object TerminalLayout {
   implicit class ColorString(msg: String) {
     def colorizeUuids: String = {
       "UUID\\((.*?)\\)".r.findAllMatchIn(msg).foldLeft(msg) {
-        case (l, r) =>
-          val color = if (Option(System.getProperty("RAINBOW_UUID")).isDefined)
-            Math.abs(17 * r.group(1).substring(0,8).map(_.toInt).product) % 209 + 22
-          else 2
```

Comment:
Oh I was just too lazy to close this. 

The background was the other day I stumbled across this and said "WTF do these numbers mean" and around the same time saw Li put out that fansi library and it seemed like a good 5 min spare time project. It turns out that a lot of the way we're highlighting doesn't fit iwth all of the sugar fansi gives you anyways. In this particular case i'm not inclined to figure out how to shoehorn RAINBOW_UUID into fansi.

I had no idea what RAINBOW_UUID was until miguel told me it was something @scottfrazer used. It wasn't documented anywhere and I even googled the term thinking perhaps it was a standard thing.